cmd/clair-load-test/clair-load-test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/quay/zlog"
+	"github.com/rs/zerolog"
+	"github.com/urfave/cli/v2"
+	"github.com/vishnuchalla/clair-load-test/pkg/reports"
```

Comment:
Acknowledged. I will change it at the end just before merged into main. Because changing it before will get us errors due to this new code structure not being present in the previous code.

Reply:
I believe it doesn't work now because you also changed the go.mod "module" directive from which the import path is dictated. Changing everything to github.com/quay/clair-load-test should be good